### DISCUSSION TITLE
None of the unique items (such as the Boom Stick) are accessible through submarine editor's item menu

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#13798] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

All unique items that can't be purchased in campaign's stores and can only be found as loot in wrecks and outposts are hidden in the submarine editor and cannot be manually placed. 
Those items are: Prototype Steam Cannon, Deadeye Carbine, Boom Stick, Europan Handshake, Mother's Providence, Mother's Countenance.

Placing unique items in a sub file can be useful for adventure maps and for testing purposes.

### Reproduction steps

1. Open the submarine editor.
2. Search for any of the items listed above.
3. No results.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.3.0.4

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Actually you can see them with debugDraw

**Reply by NotWendy** [reply]
You can see everything with debugdraw, including things like active projectiles. My question is why regular items, which can be acquired by players in the game aren't shown, based on some arbitrary limitation. Who builds with debudraw on?

**Comment by Regalis11** [comment]
This is actually intentional. The reasoning is that these should not be placed in submarines by default, they're intended to be rare rewards you might run into during the course of the campaign.
